codes/ruby/chapter_array_and_linkedlist/array.rb

```diff
@@ -0,0 +1,105 @@
+=begin
+File: array.rb
+Created Time: 2024-03-18
+Author: Xuan Khoa Tu Nguyen (ngxktuzkai2000@gmail.com)
+=end
+
+# 随机访问元素
```

Comment:
Is there another comment format to label the headings?

Reply:
> =begin 随机访问元素 =end

I see. Do you think it is a common usage? And what is the industrial standard style of heading comments for Ruby?

Reply:
> > =begin 随机访问元素 =end
> 
> I see. Do you think it is a common usage? And what is the industrial standard style of heading comments for Ruby?

Actually I barely see this usage in practice. Most of the time when I'm working with Ruby I see people prefer using the second one if they want to write a heading comment

```rb
=begin
随机访问元素
=end
```

It is clear enough to understand that they want to state a heading instead of a normal comment. If it is ok for you, I will push another commit to resolve your 2 reviews now

Reply:
I see. How about `### 随机访问元素 ###`?

Reply:
> I see. How about `### 随机访问元素 ###`? A pythonic solution

That's great! We can use it as our own Ruby heading comment convention. With this format, we can even remain compatible with the whole Python sample code.

I would like to briefly summarize the incoming Ruby comment conventions for the project:
- Heading comment: `### Heading ###`
- Inline comment: `#`
- Multiline comment:
```rb
# Abc
# Xyz
# Some thing
```
- File header:
```rb
=begin
File:
Created Time:
Author:
=end
```
- Classes/modules/methods comment (following [Yardoc](https://yardoc.org/guides/index.html) conventions)
```
    # Generate a linked list from an array
    #
    # @param [Array<Integer>] arr
    #
    # @return [ListNode] the root of the linked list
    def generate_from_array(arr) 
    end
```

Reply:
Great conclusion!

Reply:
By the way, we'd like to avoid using the "Classes/modules/methods comment" to reduce the lines of the code.